### ISSUE TITLE
non-redefined type features inherited from `Any` are not shown in the docs

### ISSUE THREAD

**Issue by simonvonhackewitz** [tokiwa-software/fuzion#3913]
TLDR: `prefix $` should also show up as a type feature, because `Type` inherits it from `Any` but there is not easy and good way to do this.

Features from `Any` appear as normal and as type features and if they are never redefined, their outer feature is `Any` in both cases. Therefore there is no easy way, for these features, to figure out if it is a type feature or not.

Also there is only one reference to an `AbstractFeature` for these features, yet it should show up in two places in the docs (but only if it is never redefined).

Currently this only affects `prefix $`.

There might be more information in #3716.

Some Ideas
1. Clone the AbstractFeatures that are collected as type features and set an internal variable in the clones that they are type features
2. Save the information that they were collected as type features in some external context similar to `FeatureAndOuter`. This would require a lot of changes and would make generating the docs unnecessary complicated.
